docs/content/docs/getting-started/nats-streaming/go.mod

```diff
@@ -13,9 +13,9 @@ require (
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/lib/pq v1.0.0 // indirect
 	github.com/nats-io/gnatsd v1.3.0 // indirect
-	github.com/nats-io/go-nats-streaming v0.4.4
 	github.com/nats-io/nats-server v1.4.1 // indirect
 	github.com/nats-io/nats-streaming-server v0.11.2 // indirect
+	github.com/nats-io/stan.go v0.4.4
```

Comment:
It seems that the release that changed the repo name was `v0.4.5` https://github.com/nats-io/stan.go/releases/tag/v0.4.5
Why v0.4.4? From what I saw, the name change was the only change between those two, so it makes no difference, I guess

Reply:
this is because sample project depends on release version of library and we need to update dependencies of library first